### PR TITLE
Implement minmax test script

### PR DESCRIPTION
**Pull request by laczkozsofi**

### CODE REVIEW

File: tests/test_files/minmax/minmax.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    Returns:
+        The populated namespace from the arguments
+    """
+    parser = argparse.ArgumentParser()
+    parser.prog = "Minmax"
+    parser.description = ("This program can select the maximum or the minimum from the given numbers."
+                          "Numbers given from the user as the command line arguments. The program select maximum by "
+                          "default, but user can ask minimum by --min flag. The result is printed to the console.")
+    parser.add_argument("--min", action="store_true", help="Whether user wants minimum selection")
```
Comment (CsullogBeni) [inline tests/test_files/minmax/minmax.py:16]: Looks great, but try to add a second name, like:
("--min", "-m",...

File: tests/test_files/minmax/minmax.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if args.min:
+        print("Minimum of the numbers:", min(args.numbers))
+    else:
+        print("Maximum of the numbers:", max(args.numbers))
+
+
+if __name__ == "__main__":
+    """call the parser and the minmax_selection method"""
+    parsed_args = pars_arguments()
+    minmax_selection(parsed_args)
```
Comment (CsullogBeni) [inline tests/test_files/minmax/minmax.py:40]: Try to reformat the file to clear warnings

File: tests/test_files/minmax/minmax.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import argparse
+
+
+def pars_arguments():
```
Comment (CsullogBeni) [inline tests/test_files/minmax/minmax.py:4]: typeo, pars should be parse

**Review by CsullogBeni — CHANGES_REQUESTED** [review]
Looks good, but minor changes needed.

File: tests/test_files/minmax/minmax.py (hunk 2 above)
Comment (laczkozsofi) [reply tests/test_files/minmax/minmax.py:40]: A reformated it. A don't know what you still see

File: tests/test_files/minmax/minmax.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    if args.min:
+        print("Minimum of the numbers:", min(args.numbers))
+    else:
+        print("Maximum of the numbers:", max(args.numbers))
+
+
+if __name__ == "__main__":
+    """call the parser and the minmax_selection method"""
+    parsed_args = parse_arguments()
+    minmax_selection(parsed_args)
```
Comment (CsullogBeni) [inline tests/test_files/minmax/minmax.py:40]: At the end of the file, a blank line should be inserted.

**Review by CsullogBeni — CHANGES_REQUESTED** [review]

File: tests/test_files/minmax/minmax.py (hunk 4 above)
Comment (CsullogBeni) [reply tests/test_files/minmax/minmax.py:40]: Yeah, thats correct, the github just does not show it, Thanks!

**Review by CsullogBeni — APPROVED** [review]

**Pull request merged**